lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -166,7 +166,7 @@ def const_name(node)
           # rubocop:enable InternalAffairs/NodeDestructuring
           if !namespace
             [name]
-          elsif namespace.cbase_type?
```

Comment:
perhaps you should check for a `const_type?` in the first place - at least that is what Node's method do

Reply:
Agree, it also makes it more clear when which branch is being used.